tsfc/tsfc_to_loopy.py

```diff
@@ -245,6 +262,9 @@ def flex_idx_to_loopy(f):
 
         return result
 
+    if c.shape == (None, ):
```

Comment:
That's a weird shape value. Where does that come from?

Reply:
This is obtained when a `Constant` is obtained in the `form`. Something like `g=9.81` in the form would be interpreted as a `FlexiblyIndexedVariable` with shape `(None, )`. And so, in the loopy kernel I do not index it, it stays un-indexed.

I will add a comment to reflect about this behvior, as this behavior is a bit confusing.

Reply:
Thanks for explaining. I was confused because the conventional shape for scalars (in numpy and loopy e.g.) is `()`, i.e. the empty tuple.